src/Microsoft.SqlTools.ServiceLayer/ShowPlan/ShowPlanGraphUtils.cs

```diff
@@ -85,5 +90,40 @@ private static List<ExecutionPlanGraphPropertyBase> GetProperties(PropertyDescri
             }
             return propsList;
         }
+
+        private static List<ExecutionPlanRecommendation> ParseRecommendations(ShowPlanGraph.ShowPlanGraph g, string fileName)
+        {
+            if (g.Description != null && g.Description.MissingIndices != null)
```

Comment:
`g.Description?.MissingIndices != null`

But also - can this ever be null currently? We're always setting a list, it just might be empty.